Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/tests-dashboard/delete-test-dialog.component.ts

```diff
@@ -32,6 +34,9 @@ export class DeleteTestDialogComponent {
             this.snackBar.open(this.successMessage, 'Dismiss', {
                 duration: 3000,
             });
+            let url = window.location.pathname;
+            if (url.includes('view'))
+            this.router.navigate(['/tests']);
```

Comment:
formatting with if-condition is not proper. what is the purpose of this condition?